### PR TITLE
parse blob object

### PR DESCRIPTION
**Pull request by ithink20**

### CODE REVIEW

File: hello.go [hunk 1]
```diff
@@ -19,6 +19,13 @@ type objectHeader struct {
 	length     int
 }
 
+type commitHeader struct {
```
Comment (anoopc1) [inline hello.go:22]: this should be called `commitObject`

File: hello.go [hunk 2]
```diff
@@ -75,6 +82,40 @@ func parseObjectHeader(bufScanner *bufio.Scanner) objectHeader {
...
+	fmt.Print(fileMetadataString)
+}
+
+func printBlobContent(bufScanner *bufio.Scanner, byteCount int) {
+	//format:
+	// <content>
+	// ...
+	// print 3KB size (atmax) of object content
+	var count int
+	if (byteCount > 3072) {
```
Comment (anoopc1) [inline hello.go:106]: declare `3072` as a constant at the top the file or function instead of duplicating at 3 places.

File: hello.go [hunk 3]
```diff
@@ -75,6 +82,40 @@ func parseObjectHeader(bufScanner *bufio.Scanner) objectHeader {
...
+		count = 3072
+	} else {
+		count = byteCount
+	}
+	fileMetadataBytes := scanCountBytes(bufScanner, count, true)
+	fileMetadataString := string(fileMetadataBytes)
+	fmt.Print(fileMetadataString)
+	if (byteCount > 3072) {
+		fmt.Printf("3KB truncated...\n")
```
Comment (anoopc1) [inline hello.go:115]: `(... truncated to 3KB)`

**Review by anoopc1 — APPROVED** [review]
LGTM

File: hello.go (hunk 3 above)
Comment (ithink20) [reply hello.go:115]: I'll update the changes.

**Pull request merged**